_docs/reference/os_support.md

```diff
@@ -0,0 +1,129 @@
+| Ubuntu 16.04/18.04 AWS     | No           | There is no linux-image-extra candidate for AWS optimised kernel   |
+| Ubuntu 16.04/18.04 Azure   | No           | There is no linux-image-extra candidate for Azure optimised kernel |
+| Ubuntu 16.04 GCE           | Yes          | Requires installation of linux-image-extra-$(uname -r) package     |
+| Ubuntu 18.04 GCE           | No           | There is no linux-image-extra candidate for GCE optimised kernel   |
+| Debian 9                   | Yes          |                                                                    |
+
+
+## How to enable LIO support
+
+The following script will try to load the modules in your host. If it finishes without error, your OS is compatible and LIO support will be enabled.
+
+It requires privileged permissions.
+
+```
+cat <<'END' > enable_lio.sh
+
+#!/bin/bash
+set -e
+
+# Configfs can be built in the kernel, hence the module
+# initstate file will not exist. Even though, the mount
+# is present and working
+echo "Checking configfs"
+if mount | grep -q "^configfs on /sys/kernel/config"; then
+    echo "configfs mounted on sys/kernel/config"
+else
+    echo "configfs not mounted, checking if kmod is loaded"
+    state_file=/sys/module/configfs/initstate
+    if [ -f "$state_file" ] && grep -q live "$state_file"; then
+        echo "configfs mod is loaded"
+    else
+        echo "configfs not loaded, executing: modprobe -b configfs"
+        modprobe -b configfs
+    fi
+
+    if mount | grep -q configfs; then
+        echo "configfs mounted"
+    else
+        echo "mounting configfs /sys/kernel/config"
+        mount -t configfs configfs /sys/kernel/config
+    fi
+fi
+
+# Enable a mod if not present
+# /sys/module/$modname/initstate has got the word "live"
+# in case the kernel module is loaded and running 
+for mod in target_core_mod tcm_loop target_core_file; do
+    state_file=/sys/module/$mod/initstate
+    if [ -f "$state_file" ] && grep -q live "$state_file"; then
+        echo "Module $mod is running"
+    else 
+        echo "Module $mod is not running"
+        echo "executing modprobe -b $mod"
+        modprobe -b $mod
+    fi
+done
+
+# Check if the modules loaded have got its
+# directories available on top of configfs
+target_dir=/sys/kernel/config/target
+core_dir="$target_dir"/core
+loop_dir="$target_dir"/loopback
+
+[ ! -d "$target_dir" ] && echo "$target_dir doesn't exist" && exit 1 # Should exist from enabling module
+[ ! -d "$core_dir" ]   && echo "$core_dir doesn't exist"   && exit 1
+[ ! -d "$loop_dir" ]   && echo "$loop_dir doesn't exist. Creating dir manually..." && mkdir $loop_dir
+
+echo "LIO set up is ready!"
+
+END
+
+chmod +x enable_lio.sh
+sudo ./enable_lio.sh
+```
+
+## Ubuntu generic and GCE
+
+It is required to install linux-image-extra for any generic ubuntu. 
+
+Google Cloud Engine has got a candidate to install linux-image-extra with gce optimisation for Ubuntu 16.04. However, Ubuntu 18.04 doesn't have a package for it yet.
```

Comment:
"has got" => "has"
". However, Ubuntu 18.04 doesn't have a package for it yet." => "but this package is not yet available for Ubuntu 18.04."

Reply:
Thank you! Done https://github.com/storageos/storageos.github.io/pull/148/commits/85f8b90c9002e67efe6eff881dff4b73ed98fad3